scripts/bootstrap.sh.in

```diff
@@ -182,43 +182,82 @@ orchestrate_bootstrap() {
     # Grains must be set (in `/etc/salt/grains`) *before* invoking `salt-call`,
     # otherwise grains set during execution won't be taken into account
     # properly.
-    ${SALT_CALL} --local --retcode-passthrough state.sls \
+    ${SALT_CALL} --local --state-output=mixed --retcode-passthrough state.sls \
         metalk8s.node.grains \
         saltenv=metalk8s-@@VERSION \
         pillarenv=metalk8s-@@VERSION
 
-    # TODO: Registry IP should not need a pillar override here
-    local -r registry_ip=$(
+    # TODO: FIX registry IP
```

Comment:
I think the removed comment is the 'more correct' one, no?

Reply:
True I will